pom.xml

```diff
@@ -950,6 +964,20 @@
                 </plugin>
 
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-dependency-plugin</artifactId>
+                    <!--
+                    Annotations are apparently the canonical use for this:
+                    http://maven.apache.org/components/plugins/maven-dependency-plugin/examples/exclude-dependencies-from-dependency-analysis.html
+                    -->
+                    <configuration>
```

Comment:
I originally had the test/demo code in presto-tests, which depended on the processor for compile, but not at runtime. This trips up the dependency checker.

Now that I've moved the test/demo code to the same package, this requirement has disappeared. I've removed this for now, and I'll see what makes sense as I go and try to come up with a more targeted solution.